src/analyses/variable-sensitivity/abstract_object_set.cpp

```diff
@@ -37,7 +37,7 @@ void abstract_object_sett::output(
   join_strings(out, output_values.begin(), output_values.end(), ", ");
 }
 
-abstract_object_pointert abstract_object_sett::to_interval()
+constant_interval_exprt abstract_object_sett::to_interval() const
 {
```

Comment:
Rather than using `to_constant` this should probably call `to_interval` so that if it is a value set of intervals it winds up taking the join of all of the intervals.

Reply:
Ah; ignore this, I have found the later commit that does this!